modules/light-clients/06-solomachine/client_state.go

```diff
@@ -125,6 +125,10 @@ func (cs *ClientState) VerifyMembership(
 		return sdkerrors.Wrap(commitmenttypes.ErrInvalidProof, "failed to unmarshal path into ICS 23 commitment merkle path")
 	}
 
+	if len(merklePath.String()) == 0 {
+		return sdkerrors.Wrap(commitmenttypes.ErrInvalidProof, "path is empty")
+	}
```

Comment:
Should we be using the `merklePath.Empty()` method which returns a `bool` instead?

Reply:
ahaa, didnt realise that was a thing. thanks!